src/annotator/plugin/toolbar.coffee

```diff
@@ -26,6 +27,15 @@ module.exports = class Toolbar extends Plugin
       $(@options.container).append @toolbar
     else
       $(@element).append @toolbar
+    
+    # https://h.readthedocs.io/projects/client/en/latest/publishers/config/?highlight=onLoginRequest#cmdoption-arg-showhighlights
+    highlights = true; # default to on
```

Comment:
The meaning of the variable `highlights` is a little unclear here—as I parse this, it's used later as a Boolean to determine whether the highlights are currently visible? Perhaps being more verbose, e.g., `highlightsAreVisible`?

Reply:
sounds good